nibabel/bv.py

```diff
@@ -126,67 +112,91 @@ def parse_BV_header(hdr_dict, fileobj):
 
     Parameters
     ----------
-    hdrDict: OrderedDict
-       Pre-populated hdrDict that contains the fields to expect for the
-       respective BV file format.
+    hdr_dict_proto: tuple
+        tuple of format described in Notes below.
     fileobj : fileobj
        File object to use. Make sure that the current position is at the
        beginning of the header (e.g. at 0).
+
+    Notes
+    -----
+    The description of `hdr_dict_proto` below is notated according to
+    https://docs.python.org/3/reference/introduction.html#notation (except
+    using double parens indicating a group, single parens indicating a tuple)::
+
+        element_proto ::= (name, pack_format, default)  |
+                        (name, proto, n_fields_name)
+        proto ::= ((element_proto))*
```

Comment:
Shouldn't this be ```hdr_dict_proto ::= ((element_proto))*```?
Also, maybe this should be the first line of the notation (_i.e._ switch with line 127/128)?